packages/react-charting/jest.config.js

```diff
@@ -10,8 +10,8 @@ function getEsmOnlyPackagesToCjsMapping() {
    */
   const cjsPathsToEsmOnlyPackages = {
     '^d3-scale$': prefix + 'node_modules/d3-scale/dist/d3-scale.js',
-    '^d3-array$': prefix + 'node_modules/d3-scale/node_modules/d3-array/dist/d3-array.js',
-    '^d3-time$': prefix + 'node_modules/d3-scale/node_modules/d3-time/dist/d3-time.js',
+    '^d3-array$': prefix + 'node_modules/d3-array/dist/d3-array.js',
+    '^d3-time$': prefix + 'node_modules/d3-time/dist/d3-time.js',
```

Comment:
these two can be removed as they properly reference CJS modules via `pacakge.json#main`. it will also serve us as good integration test if those packages do some unexpected changes to their public API surface

```suggestion
```

Reply:
Done